packages/x-components/src/x-modules/facets-next/entities/editable-number-range-filter.entity.ts

```diff
@@ -96,8 +95,8 @@ export class EditableNumberRangeFilterEntity implements FilterEntity {
    * @internal
    */
   protected getFilterByFacet(facetId: Facet['id']): EditableNumberRangeFilter | undefined {
-    return (
-      this.store.getters['x/facetsNext/filtersByFacet'] as FacetsNextGetters['filtersByFacet']
-    )[facetId]?.[0] as EditableNumberRangeFilter;
+    return Object.values(this.store.state.x.facetsNext.filters).find(
+      filter => isEditableNumberRangeFilter(filter) && filter.facetId === facetId
```

Comment:
Why not just using the new `facets` getter instead of the find loop?

Reply:
This was because the problem we talk, that doesn't exist the Facet of the EditableNumberRangeFilter. With the new change of the entity, that adds the facet in case it doesn't exist, we can now use the `facets` getter here.